assets/js/components/settings/SettingsActiveModule/Footer.js

```diff
@@ -72,6 +73,9 @@ export default function Footer( props ) {
 	const isSaving = useSelect( ( select ) =>
 		select( CORE_UI ).getValue( isSavingKey )
 	);
+	const accountChooserBaseURI = useSelect( ( select ) =>
+		select( CORE_USER ).getAccountChooserURL( module.homepage )
+	);
```

Comment:
We should consider the fact that `module` could be `undefined`, or `module.homepage` could be undefined or an empty string at this stage. Otherwise, the check [here](https://github.com/google/site-kit-wp/pull/5984/files#diff-35a209af9b8af36a7078eba599222127fa6ef4c9bdaa8b38b5dfecfdd685a2ceR220) wouldn't make sense because `getAccountChooserURL` would always return a truthy value.

I would recommend returning `undefined` from this hook if:
- `module` is falsy.
- `module.homepage` is falsy, or is an empty string (you can use `isEmpty()` from `lodash`.

I would also recommend changing the name of this variable to something like `moduleHomepage`.

Reply:
@nfmohit makes sense, thank you.

Reply:
Thank you, @sashadoes! [I have updated the condition](https://github.com/google/site-kit-wp/pull/5984/commits/f1de916b2570ebbe1524a9a175052cc98d4869cf) a little to take an empty string for `module.homepage` into consideration, based on [this parameter](https://github.com/google/site-kit-wp/blob/6590c4ecd41ef6018d2f243983e1974c3b82dc24/assets/js/googlesitekit/modules/datastore/modules.js#L329).